tinycrypt/ecc_dsa.c

```diff
@@ -301,7 +303,18 @@ int uECC_verify(const uint8_t *public_key, const uint8_t *message_hash,
 	}
 
 	/* Accept only if v == r. */
-	return (int)(uECC_vli_equal(rx, r) == 0);
+	diff = uECC_vli_equal(rx, r);
+	if (diff == 0) {
+		mbedtls_platform_enforce_volatile_reads();
+		if (diff == 0) {
```

Comment:
It think it is. Our reference document for FI countermeasures also checks the same statement twice, rather than a statement and its negation. Perhaps more importantly, I've reviewed the generated assembly and it does include sufficient checks.